tests/vault/test_vault_asymmetric.yml

```diff
@@ -14,18 +14,101 @@
       ipaadmin_password: SomeADMINpassword
       name: asymvault
       vault_type: asymmetric
-      public_key: "{{ lookup('file', 'public.pem', rstrip=False) | b64encode }}"
+      public_key: "{{ lookup('file', 'old_public.pem', rstrip=True) | b64encode }}"
```

Comment:
Please do not use the b64encode filter any more. We already removed it everywhere else.

Reply:
Done.